lib/Windowing/README.MD

```diff
@@ -0,0 +1,124 @@
+# Windowing
+
+A selection of components to give you virtualisation.
+
+## How does it work?
+
+A `div` is rendered that is high enough to hold every item, it is simply calculated as follows:
```

Comment:
Some content change suggestions;

- use "tall" rather than "high"
- remove the word simply 😄